src/Http/Controllers/VoyagerDatabaseController.php

```diff
@@ -309,6 +309,36 @@ public function updateBread(Request $request, $id)
         }
     }
 
+    public function addBreadRelationship(Request $request, $id)
+    {
+        Voyager::canOrFail('browse_database');
```

Comment:
I'm not sure this is the correct permission to use here.  Is this an artifact from you copy/pasting?